dsc_lib/src/configure/mod.rs

```diff
@@ -246,7 +247,8 @@ impl Configurator {
             result.results.push(resource_result);
         }
 
-        pb.finish_with_message("Get configuration completed");
+        mem::drop(pb_span_enter);
```

Comment:
is this needed explicitly?  I would expect both to be dropped when going out of scope

Reply:
it's needed within the config functions but I removed it from resource list